cmd/kubeadm/test/token_test.go

```diff
@@ -29,12 +29,20 @@ const (
 )
 
 var kubeadmPath string
+var testCmd = true
 
 func init() {
 	flag.StringVar(&kubeadmPath, "kubeadm-path", filepath.Join(os.Getenv("KUBE_ROOT"), "cluster/kubeadm.sh"), "Location of kubeadm")
+	if _, err := os.Stat(kubeadmPath); os.IsNotExist(err) {
```

Comment:
this really should not be in an init() block... make it a helper that each test calls. global flag registration also does not belong in an init() block of a test file, but that can be a separate issue